packages/plugin-json/examples/schema.graphql

```diff
@@ -17,6 +17,11 @@ type Test @amplienceContentType {
 type Test2 @amplienceContentType {
   name: String
   test: Test
+  extension: Test3 @amplienceExtension(name: "extension-name")
```

Comment:
Very nice of you to update the example :)
Could you also update the tests for this feature?
You can update base.graphql file and the expected json inside the testfolder (run pnpm test to verify)